xero_files.yaml

```diff
@@ -909,6 +943,10 @@ components:
           $ref: '#/components/schemas/ObjectGroup'
         ObjectType:
           $ref: '#/components/schemas/ObjectType'
+    AssociationCount:
+      type: object
+      additionalProperties:
+        type: integer       
```

Comment:
Delete 946-949. Since we no longer $ref a component schema when describing the response type as an object, this AssociationCount schema isn't being consumed anywhere